README.md

```diff
@@ -9,10 +9,9 @@ Here's a list of the examples in this repository.
 If you're new to Blocky AS, we recommend working though these examples in the
 following order:
 
-- [Hello World - Attesting a Function Call](./hello_world_attest_fn_call) shows
+- [Attesting a Function Call](attest_fn_call) shows
```

Comment:
Since we're moving towards primitive examples that show small chunks of functionality, I don't think we need this hello world part to indicate this example is the starting point.